### ISSUE TITLE
accuracy problem

### ISSUE THREAD

**Issue by MS-MA** [Bartzi/see#76]
hello Bartzi,sorry, I am bothering you again.
What is the role of train-accuracy and validation-accuracy when training a model with the fsns dataset? (a word or a picture of all words or other)

**Comment by Bartzi** [comment]
`train-accuracy` is the accuracy of the model on the training set.
`validation-accuracy` is the accuracy of the model on the validation set.

**Comment by MS-MA** [comment]
I understand these. What I want to know is how the accuracy is calculated, that is, its calculation formula.

**Comment by MS-MA** [comment]
For example, this form of calculation formula, the number of correctly recognized words in the training set divided by the number of words included in the training set

**Comment by Bartzi** [comment]
We use the standard calculation formula, number of correctly recognized words divided by the number of all words.
